.github/dependabot.yml

```diff
@@ -9,3 +9,7 @@ updates:
       - dependabot
     commit-message:
       prefix: BAU
+  - package-ecosystem: "gradle" 
+    directory: "/" 
+    schedule:
+      interval: "daily"
```

Comment:
is there a requirement for this to be daily otherwise can we make it weekly so it's consistent with everything else please? thanks

Reply:
Good point - done